Components/Metrics/SumOfPairwiseCorrelationsMetric/itkSumOfPairwiseCorrelationCoefficientsMetric.h

```diff
@@ -167,7 +167,7 @@ class ITK_TEMPLATE_EXPORT SumOfPairwiseCorrelationCoefficientsMetric
   FixedImageSizeType m_GridSize{};
 
   /** Bool to indicate if the transform used is a stacktransform. Set by elx files. */
-  bool m_TransformIsStackTransform{ true };
+  bool m_TransformIsStackTransform{ false };
```

Comment:
This commit just changes the initial value of m_TransformIsStackTransform, for SumOfPairwiseCorrelationCoefficientsMetric. Without this fix, I think the `if (!this->m_TransformIsStackTransform)` branch cannot be reached:

https://github.com/SuperElastix/elastix/blob/4f35e6fb2b21bde7903ade3266a5658cd8471268/Components/Metrics/SumOfPairwiseCorrelationsMetric/itkSumOfPairwiseCorrelationCoefficientsMetric.hxx#L486-L491

@stefanklein Do you think this fix is OK? Or would it be preferable to just assume that m_TransformIsStackTransform is always true, for this particular metric?